deeprank2/utils/parsing/__init__.py

```diff
@@ -94,7 +98,11 @@ def get_charge(self, atom: Atom) -> float:
                     charge = float(action["CHARGE"])
 
         if charge is None:
-            _log.warning(f"Atom {atom} is unknown to OPLS, the forcefield used in deeprank2; charge is set to 0.0")
+            _log.warning(
+                f"Atom {atom} is unknown to the forcefield, charge is set to 0.0. \n \
+                    This will affect `electrostatic` and `atom_charge` features.\n \
+                        Check https://deeprank2.readthedocs.io/en/latest/features.html#nonbond-energies for more details.",
```

Comment:
```suggestion
                f"Atom {atom} is unknown to the forcefield, `electrostatic` and `atom_charge` charge is set to 0.0.\n"
                "   This will affect `electrostatic` and `atom_charge` features.\n"
                "       Check https://deeprank2.readthedocs.io/en/latest/features.html#nonbond-energies for more details.",
```